src/libraries/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -1064,6 +1064,8 @@ string Stream2String(Stream s)
             0x4c, 0xb9, 0x7d, 0x87, 0xc6, 0x2b, 0xf0, 0x41, 0xee, 0x6c, 0xba,
             0x08, 0xcf, 0x69, 0x31, 0x18 };
 
+        private const string _pkcs12Password = "mono";
```

Comment:
Why doesn't this one need to use TestSecretHelper?

If it's const-able:

```suggestion
        private const string Pkcs12Password = "mono";
```

Reply:
Honestly the tool rules are not well predictable. I expect it would be reported if it is `private const string password = "mono"`

The `_` prefix is there to be consistent wit other fields

Reply:
`const` fields don't get an underscore